pkg/watcher/reconciler/pipelinerun/pipelinerun.go

```diff
@@ -30,85 +29,53 @@ import (
 )
 
 type Reconciler struct {
-	logger            *zap.SugaredLogger
-	client            pb.ResultsClient
+	client            *results.Client
 	pipelineclientset versioned.Interface
 }
 
 func (r *Reconciler) Reconcile(ctx context.Context, key string) error {
-	logger := logging.FromContext(ctx)
+	log := logging.FromContext(ctx)
+	log.With(zap.String("key", key))
 
 	namespace, name, err := cache.SplitMetaNamespaceKey(key)
 	if err != nil {
-		logger.Errorf("Invalid resource key: %s", key)
+		log.Errorf("invalid resource key: %s", key)
 		return nil
 	}
-	logger.With(zap.String("Namespace", namespace), zap.String("Name", name))
 
-	pr, err := r.pipelineclientset.TektonV1beta1().PipelineRuns(namespace).Get(name, metav1.GetOptions{})
+	// Lookup current PipelineRun.
+	tr, err := r.pipelineclientset.TektonV1beta1().PipelineRuns(namespace).Get(name, metav1.GetOptions{})
 	if errors.IsNotFound(err) {
-		logger.Errorf("PipelineRun in work queue no longer exists: %v", err)
+		log.Warnf("TaskRun not found: %v", err)
```

Comment:
It should be PipelineRun?

Reply:
I did catch these but pushed them to a wrong branch. 🙃 
 Fixed.